RadixWallet/Features/AppFeature/Overlay/FullScreenOverlayCoordinator+Reducer.swift

```diff
@@ -0,0 +1,63 @@
+import ComposableArchitecture
+import SwiftUI
+
+public struct FullScreenOverlayCoordinator: Sendable, FeatureReducer {
+	public struct State: Sendable, Hashable {
+		public var root: Root.State
+
+		public init(root: Root.State) {
+			self.root = root
+		}
+	}
+
+	public enum ChildAction: Sendable, Equatable {
+		case root(Root.Action)
+	}
+
+	public enum DelegateAction: Sendable, Equatable {
+		case dismiss
+	}
+
+	public struct Root: Sendable, Hashable, Reducer {
+		@CasePathable
+		public enum State: Sendable, Hashable {
+			case claimWallet(ClaimWallet.State)
+		}
+
+		@CasePathable
+		public enum Action: Sendable, Equatable {
+			case claimWallet(ClaimWallet.Action)
+		}
+
+		public var body: some ReducerOf<Self> {
+			Scope(state: /State.claimWallet, action: /Action.claimWallet) {
```

Comment:
Please use the new version of these:
`Scope(state: \.claimWallet, action: \.claimWallet)`